### ISSUE TITLE
Great Projects

### ISSUE THREAD

**Issue by PanzerfaustJoe** [PhelanBavaria/ancienttimeline#51]
I see that a new system for great projects or "wonders" has been added. Like I mentioned months agom I think they should work somewhat like Civilization's system, taking awhile to build but granting great permanent bonuses to the builder. I figured I should open up an issue so we can discuss which ones to add. Here is my list so far, more will be added later. I will make bonuses later as well.

Hanging Gardens of Babylon
Colossus of Rhodes
Lighthouse of Alexandria
Circus Maximus
Great Wall of China
Parthenon
Tower of Babel?

**Comment by Firesoul7** [comment]
The Tower of Babel is a Bible story, when all men spoke one language in the
ancient times, they came together and built a vast tower to travel freely
between heaven and earth. God didn't like this and so stuck the tower down,
and made every man speak a different language so they could never do the
same again. Suffice it to say, not a great thing to be building. You could
replace it with the Library of Ashurbanipal though (I may have spelled that
incorrectly)

**Comment by PanzerfaustJoe** [comment]
I am quite familiar with the story. However, some historians and archaeologists believe that the tower did in fact exist at some point, just like how some found what they believe to be the remnants of Noah's ark. Since their is no absolute proof as to whether it existed or didn't, that factor does not matter. I do not think it should be excluded just because it was a bad idea, because it was. The guy who ordered it constructed was named "Nimrod" so that explains a lot :D

However, it still is a wonder to behold nonetheless. I do agree with the Library of Ashurbanipal.

**Comment by qweytr** [comment]
For now there's not much point to implement wonders that were built before the timeline of the initial release.

**Comment by Science-Recon** [comment]
The library of Alexandria might be one worth doing. Also, Alexandria should not be called Alexandria at the start. For this I'm thinking maybe an event that would rename the city to Alexandria and then you as Macedonia have the option to build the library or not. (As opposed to it being a decision)

**Comment by PhelanBavaria** [comment]
Didn't we have an issue on this matter already?

**Comment by Firesoul7** [comment]
Yes, for some reason another was opened and the original closed. Maybe
because there was an update? But then with an update it would have been
better to update the old issue.
On 20 Apr 2016 18:19, "PhelanBavaria" notifications@github.com wrote:

Didn't we have an issue on this matter already?

**Comment by Science-Recon** [comment]
Is it possible to edit/add animated map objects. (E.g. The pyramids and the animals)?

**Comment by qweytr** [comment]
I haven't done anything to the map objects, so I have no idea how much they can be edited, but I'd assume editing or adding them would be very difficult and there are more important things to focus on.

**Comment by Science-Recon** [comment]
Well, I think that they're just 3D sprites, however yes, that is something to think about much later.

**Comment by Science-Recon** [comment]
If we do get round to this, should be quite doable now with the new exporter.

**Comment by qweytr** [comment]
Does anyone in the team have Maya?

**Comment by Science-Recon** [comment]
Hmm.. I have maya, but have so far been unsuccessful in actually doing anything with it; graphic design isn't my forte.
